logging/build.gradle.kts

```diff
@@ -79,6 +79,8 @@ kotlin {
         }
         val jvmMain by getting {
             dependencies {
+
+                // `FluentLogger2` is exposed by the deprecated `Logging` interface.
```

Comment:
Please add more context details here. It's difficult to comprehend without being deep into the problem.